data/elevation-and-terrain/index.html

```diff
@@ -23,7 +23,7 @@
       label: 'Download: Statewide 30 meter USGS DEM'
   updates:
     - January 2018
----
+----
```

Comment:
```suggestion
----
```
```suggestion
---
```
Borks page header?

Reply:
And it looks like the downloads are gone from the USGS 3DEP section on this page (data/elevation-and-terrain/index.html)